com.unity.probuilder/Editor/MenuActions/Export/ExportAsset.cs

```diff
@@ -217,13 +205,32 @@ static string ExportMesh(string path, ProBuilderMesh mesh)
             mesh.ToMesh();
             mesh.Refresh();
             mesh.Optimize();
-            mesh.mesh.name = name;
 
-            string meshPath = AssetDatabase.GenerateUniqueAssetPath(string.Format("{0}/{1}.asset", relativeDirectory, name));
+            var meshAsset = Object.Instantiate(mesh.mesh);
```

Comment:
This breaks the `Replace Source` option. When exporting as an asset with `Replace Source` enabled, the `ProBuilderMesh` component should be removed from the `GameObject` and it's `MeshFilter.sharedMesh` should point to the saved asset.

Reply:
Fixed